### PR TITLE
Update multi account code to use single cur

### PR DESCRIPTION
**Pull request by parth-nclouds**
- separate Cost Usage Report creation for master payer account and nOps iam role child account.

### CODE REVIEW

File: nops-aws-account-register/README.md [hunk 1]
```diff
@@ -14,6 +14,9 @@
 ### cloudformation-single-acc-register
 This folder contains CloudFormation template to register a single AWS account with nOps.
 
+### cloudformation-org-accounts-register
+This folder contains CloudFormation template to register AWS accounts with consolidated billing and organisation to nOps.
```
Comment (gkarmark) [inline nops-aws-account-register/README.md:18]: This folder contains the CloudFormation template to register AWS accounts with consolidated billing and organization to nOps.

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf [hunk 2]
```diff
@@ -45,4 +45,14 @@ variable "AccNameToRegister" {
...
+}
+
+variable "org_root_id" {
+  type        = list(string)
+  description = "AWS Organisation root id."
+}
+
+variable "master_payer_acc_id" {
+  type        = string
+  description = "AWS org Consolidted billing master payer account id."
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: change Consolidted to: Consolidated

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf [hunk 3]
```diff
@@ -45,4 +45,14 @@ variable "AccNameToRegister" {
 variable "list_of_OU_ids" {
   type        = list(string)
   description = "list of organisation unit ids to deploy stacks."
-}
\ No newline at end of file
+}
+
+variable "org_root_id" {
+  type        = list(string)
+  description = "AWS Organisation root id."
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: Organisation to: Organization

File: nops-aws-account-register/README.md (hunk 1 above)
Comment (gkarmark) [inline nops-aws-account-register/README.md]: Change to: This folder contains CloudFormation templates to register multiple AWS accounts within an organization, for consolidated billing. 
Change organisation to: **organization**

**Review by gkarmark — COMMENTED** [review]
Parth, see inline. Can you ping me when you are on so I can figure out the correct sentence. thanks

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf (hunk 2 above)
Comment (parth-nclouds) [reply nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: done

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf (hunk 3 above)
Comment (parth-nclouds) [reply nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: done

File: nops-aws-account-register/README.md (hunk 1 above)
Comment (parth-nclouds) [reply nops-aws-account-register/README.md]: done

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/README.md [hunk 4]
```diff
@@ -29,25 +31,36 @@ This Terraform module creates CloudFormation stacksets and creates resources req
...
+| <a name="input_BucketName"></a> [BucketName](#input\_BucketName) | s3 bucket name for nOps daily reports | `string` | `"nopsbucketforlogs"` | no |
+| <a name="input_ExternalId"></a> [ExternalId](#input\_ExternalId) | External id required for cross account access role for nOps | `string` | n/a | yes |
+| <a name="input_ReportName"></a> [ReportName](#input\_ReportName) | nOps daily generated report name | `string` | `"nopsbilling-daily-gzip"` | no |
+| <a name="input_child_acc_template_url"></a> [child\_acc\_template\_url](#input\_child\_acc\_template\_url) | CloudFormation template url for child accounts. | `string` | `"https://nops-register-aws-account-us-east-1.s3.amazonaws.com/child_consolidated_aws_acc_nops_register.yaml"` | no |
+| <a name="input_list_of_OU_ids"></a> [list\_of\_OU\_ids](#input\_list\_of\_OU\_ids) | list of Organization unit ids to deploy stacks on child AWS accounts. | `list(string)` | n/a | yes |
+| <a name="input_master_payer_acc_id"></a> [master\_payer\_acc\_id](#input\_master\_payer\_acc\_id) | AWS org Consolidted billing master payer account id. | `string` | n/a | yes |
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/README.md]: Line 56: I still show a typo for Consolidated. Please correct it

File: nops-aws-account-register/README.md (hunk 1 above)
Comment (gkarmark) [reply nops-aws-account-register/README.md]: The sentence needs to be changed to:
This folder contains a CloudFormation template to register multiple AWS accounts within an organization, for consolidated billing.

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/variables.tf [hunk 5]
```diff
@@ -46,13 +46,29 @@ variable "AccNameToRegister" {
...
+}
+
+variable "org_root_id" {
+  type        = list(string)
+  description = "AWS Organisation root id."
+}
+
+variable "master_payer_acc_id" {
+  type        = string
+  description = "AWS org Consolidted billing master payer account id."
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/variables.tf]: Consolidated

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/variables.tf [hunk 6]
```diff
@@ -46,13 +46,29 @@ variable "AccNameToRegister" {
...
+  default     = "https://nops-register-aws-account-us-east-1.s3.amazonaws.com/child_consolidated_aws_acc_nops_register.yaml"
+  description = "CloudFormation template url for child accounts."
 }
 
 variable "list_of_OU_ids" {
   type        = list(string)
-  description = "list of organisation unit ids to deploy stacks."
-}
\ No newline at end of file
+  description = "list of organisation unit ids to deploy stacks on child AWS accounts."
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/variables.tf]: organization

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf (hunk 2 above)
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: Line 57 Consolidated

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf (hunk 3 above)
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: Line 52: Organization

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf [hunk 7]
```diff
@@ -45,4 +45,14 @@ variable "AccNameToRegister" {
 variable "list_of_OU_ids" {
   type        = list(string)
   description = "list of organisation unit ids to deploy stacks."
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/examples/complete/variables.tf]: Organization

File: nops-aws-account-register/terraform-multi-acc-register-via-stacksets/README.md [hunk 8]
```diff
@@ -29,25 +31,36 @@ This Terraform module creates CloudFormation stacksets and creates resources req
...
+|------|-------------|------|---------|:--------:|
+| <a name="input_AccNameToRegister"></a> [AccNameToRegister](#input\_AccNameToRegister) | Account name to register in nOps. | `string` | n/a | yes |
+| <a name="input_BucketName"></a> [BucketName](#input\_BucketName) | s3 bucket name for nOps daily reports | `string` | `"nopsbucketforlogs"` | no |
+| <a name="input_ExternalId"></a> [ExternalId](#input\_ExternalId) | External id required for cross account access role for nOps | `string` | n/a | yes |
+| <a name="input_ReportName"></a> [ReportName](#input\_ReportName) | nOps daily generated report name | `string` | `"nopsbilling-daily-gzip"` | no |
+| <a name="input_child_acc_template_url"></a> [child\_acc\_template\_url](#input\_child\_acc\_template\_url) | CloudFormation template url for child accounts. | `string` | `"https://nops-register-aws-account-us-east-1.s3.amazonaws.com/child_consolidated_aws_acc_nops_register.yaml"` | no |
+| <a name="input_list_of_OU_ids"></a> [list\_of\_OU\_ids](#input\_list\_of\_OU\_ids) | list of organisation unit ids to deploy stacks on child AWS accounts. | `list(string)` | n/a | yes |
```
Comment (gkarmark) [inline nops-aws-account-register/terraform-multi-acc-register-via-stacksets/README.md]: Line 55: change organisation to organization

File: nops-aws-account-register/README.md [hunk 9]
```diff
@@ -24,6 +24,3 @@ This folder contains the python code for the lambda function which registers AWS
 This folder contains a Terraform module which deploys CloudFormation stacksets.
 CloudFormation stacksets deploys the CloudFormation template for all accounts defined in the organizational unit (OU).
```
Comment (gkarmark) [inline nops-aws-account-register/README.md:25]: deploy
not: deploys

**Review by gkarmark — COMMENTED** [review]
please make the one change

**Pull request merged**